plugins/org.jkiss.dbeaver.model.event/src/org/jkiss/dbeaver/model/websocket/event/WSEventTopic.java

```diff
@@ -20,6 +20,7 @@
 public enum WSEventTopic {
     SERVER_CONFIG("cb_config"),
     SESSION_LOG("cb_session_log"),
+    WEBSOCKET("cb_websocket"),
```

Comment:
let's call this topic as `cb_session`, in the future there also will be 'expired session event'

Reply:
Renamed